Makefile

```diff
@@ -29,6 +30,9 @@ clean:
 	$(MAKE) -C utils clean
 	$(MAKE) -C $(PYTHON_LIB_DIR) clean
 
+build-csharp:
+	$(MAKE) -C $(CSHARP_LIB_DIR) build
```

Comment:
You don't have a `build` target in your Makefile. So you might either add it there or mention the target `all` that is in the Makefile.